test/end-to-end.js

```diff
@@ -970,6 +976,44 @@ describe('end-to-end', function() {
       });
     });
 
+    it('shows error behavior responses in development mode', function(done) {
+      var app = loopback({ localRegistry: true, loadBuiltinModels: true });
+      var bootOptions = {
+        appRootDir: SANDBOX,
+        env: 'development'
+      };
+      boot(app, bootOptions, function(err) {
+        if (err) return done(err);
+        app.listen(0, function() {
+          request(app)
+            .get('/url-does-not-exist')
+            .end(function(err, res) {
+              expect(res.error.text.match(/url-does-not-exist/));
```

Comment:
Ah, is `res.error.text` something coming from supertest? I was expecting to see a check verifying result JSON body.

I am also not sure what happens when the request fails in such way that there is no `res` object to return, how does supertest handle that case. To be safe, I usually do this:

```js
request(app)
  .get('/url-does-not-exist')
  .expect(404)
  .end(function(err, res) {
    if (err) return done(err);
    // verify & done
  });
```